README.md

```diff
@@ -1,8 +1,8 @@
-# Obsidian Latex Suite <img src="https://img.shields.io/github/manifest-json/v/artisticat1/obsidian-latex-suite"> <img src="https://img.shields.io/github/downloads/artisticat1/obsidian-latex-suite/total">
+# Obsidian Latex Suite `<img src="https://img.shields.io/github/manifest-json/v/artisticat1/obsidian-latex-suite">` `<img src="https://img.shields.io/github/downloads/artisticat1/obsidian-latex-suite/total">`
```

Comment:
It looks like the formatting in the readme has gotten muddled up. Could you revert the changes to the formatting?

Reply:
Sorry for the late reply. A vscode extension I was using reformated the readme and I have restored it.